wiki/Beatmap_ranking_procedure/de.md

```diff
@@ -60,3 +60,8 @@ Beatmap-Nominatoren und NAT-Mitglieder können in Einzelfällen ein [Veto](/wiki
 Beatmaps, die die Qualifizierungsphase durchlaufen haben, haben das Ranking-Verfahren abgeschlossen. Sie haben [Ranglisten](/wiki/Ranking) und vergeben Performance-Punkte an Spieler.
 
 Bei gerankten Beatmaps wird der Ranked-Status nur in Ausnahmefällen entfernt, wenn kurz nach Abschluss des Ranking-Verfahrens Probleme entdeckt werden.
+
+## Anmerkungen
+
+[^main-game-mode]: Der Spielmodus, für den die meisten Schwierigkeitsgrade erstellt wurden, wird als "Hauptspielmodus" erachtet. Wenn 2 oder mehr Spielmodi dieselbe Anzahl an Schwierigkeitsgraden haben, ist der Spielmodus der Hauptmodus, für den der Beatmap-Ersteller die meisten Schwierigkeitsgrade erstellt hat. Falls beide Bedingungen zutreffen, ist der zuerst nominierte Modus der Hauptspielmodus.
```

Comment:
```suggestion
[^main-game-mode]: Der Spielmodus, für den die meisten Schwierigkeitsgrade erstellt wurden, wird als "Hauptspielmodus" erachtet. Wenn zwei oder mehr Spielmodi dieselbe Anzahl an Schwierigkeitsgraden haben, ist der Spielmodus der Hauptmodus, für den der Beatmap-Ersteller die meisten Schwierigkeitsgrade erstellt hat. Falls beide Bedingungen zutreffen, ist der zuerst nominierte Modus der Hauptspielmodus.
```
Not strictly required, but if we're going by [the Duden's recommendation](https://www.duden.de/sprachwissen/rechtschreibregeln/zahlen-und-ziffern), numbers with one or two syllables are generally written in letters.